src/ainft.ts

```diff
@@ -25,6 +27,11 @@ export default class AinftJs {
     this.discord = new Discord(this.ain, this.baseUrl, '/discord');
     this.event = new Event(this.ain, this.baseUrl, '/event');
     this.store = new Store(this.ain, this.baseUrl, '/store');
+    this.personaModels = new PersonaModels(
+      this.ain,
+      this.baseUrl,
+      "/personaModels"
```

Comment:
Could you update `personaModels` to `persona-models`?

Reply:
I will update. Thank you!

Reply:
@589hero (you can use 'done' rather than 'will do'. ref: https://www.notion.so/comcom/ComCom-Code-Review-Guide-514844fe00624e2486c57d5791ed9a2c)